WordPressKit/PostServiceRemoteREST.m

```diff
@@ -289,7 +289,9 @@ - (void)restorePost:(RemotePost *)post
 {
     NSParameterAssert([post isKindOfClass:[RemotePost class]]);
 
-    NSString *path = [NSString stringWithFormat:@"sites/%@/posts/%@/restore", self.siteID, post.postID];
+    // The parameters are passed as part of the string here because AlamoFire doesn't encode parameters on POST requests.
+    // https://github.com/wordpress-mobile/WordPressKit-iOS/pull/385
+    NSString *path = [NSString stringWithFormat:@"sites/%@/posts/%@/restore?context=edit", self.siteID, post.postID];
```

Comment:
📓 I added the query parameters as part of the string here because AlamoFire [doesn't parse parameters](https://github.com/Alamofire/Alamofire/blob/32573d05b91d8b7349ca31b4726e80212483f02a/Source/ParameterEncoding.swift#L298-L315) when sent to a POST request.

Reply:
I think I've run into this before, should be fine here hard-coded I think 👍. These parameters are static so there's no risk of issues regarding URL encoding.